<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushQuit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelStaticIPAddress">
         <property name="text">
          <string>IPAddress:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelIPAddress">
         <property name="text">
          <string>Null</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelStaticArea">
         <property name="text">
          <string>Area:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelArea">
         <property name="text">
          <string>Null</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelStaticNumber">
         <property name="text">
          <string>No.:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelNumber">
         <property name="text">
          <string>Null</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="labelStaticPosition">
         <property name="text">
          <string>Position:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPosition">
         <property name="text">
          <string>Null</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
